### PR TITLE
Improve file labeling in replies

### PR DESCRIPTION
**Pull request by SimonBrandner**
element-web notes: none

### CODE REVIEW

File: src/utils/FileUtils.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2021 Šimon Brandner <simon.bra.ag@gmail.com>
```
Comment (turt2live) [inline src/utils/FileUtils.ts:2]: copyright of the foundation would be maintained for the copy/paste bits, in addition to your copyright.

**Comment by SimonBrandner** [comment]
I can't even spell :sweat_smile:

**Review by t3chguy — APPROVED** [review]
Great, thanks

**Comment by t3chguy** [comment]
Needs some conflicts fixing though

**Pull request merged**